code/standards/checkstyle.xml

```diff
@@ -41,22 +47,24 @@
     <module name="MethodParamPad"/>
     <module name="NoWhitespaceAfter"/>
     <module name="NoWhitespaceBefore"/>
-    <module name="OperatorWrap"/>
     <module name="ParenPad"/>
     <module name="TypecastParenPad"/>
     <module name="WhitespaceAfter"/>
     <module name="WhitespaceAround">
       <property name="tokens" value="ASSIGN"/>
     </module>
     <module name="ModifierOrder"/>
-    <module name="AvoidNestedBlocks"/>
+    <module name="AvoidNestedBlocks">
+      <property name="allowInSwitchCase" value="true" />
```

Comment:
personal style. Mostly want to allow code like 
case 2:
{
  code here;
}

as the meta comment above notes: if the only concern here are the particular style rules just let me know what to change them to. In the general sense I tried to capture existing pcgen style.